fdbserver/workloads/FuzzApiCorrectness.actor.cpp

```diff
@@ -383,8 +383,8 @@ struct FuzzApiCorrectnessWorkload : TestWorkload {
 		ExceptionContract contract;
 		std::vector<ThreadFuture<Void> > pre_steps;
 
-		BaseTest(unsigned int id_, FuzzApiCorrectnessWorkload *wl, const char *func)
-			: id(id_), workload(wl), contract(func, std::bind(&Subclass::augmentTrace, static_cast<Subclass *>(this), ph::_1)) {}
+		BaseTest(unsigned int id_, FuzzApiCorrectnessWorkload* wl, const char* func)
```

Comment:
Oops I forgot to include the report in the commit message here. Basically the problem is that `Subclass`'s constructor hasn't returned yet, so this is not an instance of `Subclass`, so we can't cast this to `Subclass`